README.md

```diff
@@ -35,6 +35,42 @@ From there, you can use one of our technology integrations such those for
 Have fun! We have a Google Group and a Slack channel we use for both developers and 
 users. Check them out [here](https://projectnessie.org/develop/).
 
+### Authentication
+
+By default, Nessie servers run with authentication disabled and all requests are processed under the "anonymous"
+user identity.
+
+In production, Nessie supports OpenID, which can be enabled by setting the following Quarkus properties:
+* `nessie.auth.enabled=true`
+* `quarkus.oidc.auth-server-url=<OpenID Server URL>`
+
+For example, to run Nessie with Keycloak in docker first start Keycloak:
+
+```
+docker run -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin --name keycloak quay.io/keycloak/keycloak:15.0.1
```

Comment:
It's a bit confusing here. The paragraph above say "in production", but this docker-run is pretty insecure. Not sure whether we should guide people about how to run Keycloak. Is there a docs page for Keycloak about how to run it in Docker for a test drive?

Reply:
would `quay.io/keycloak/keycloak:latest` be better here?

Reply:
I did not want to use `latest` to avoid surprises when things change, but it's probably stable enough for this example... will change to `latest`

Reply:
Re: "in production". I guess this really means we do not support `Basic` auth in production. I can rephrase to that effect.

Reply:
Re: Keycloak, they certainly have "how to" docs, but I wanted to give a specific example that people can try and run. Would it be preferable to have a generic "auth URL" in the example and let them figure out how to configure the auth server? 

Another option, I suppose, is to put a specific example in docs rather than README. WDYT?

Reply:
I've pushed a new version of README in 50fd26f200704543af4418ed53349858d0fbd5b8. How does it sound?